vulnerabilities/importers/safety_db.py

```diff
@@ -92,7 +93,7 @@ def set_api(self, packages):
         asyncio.run(self._versions.load_api(packages))
 
     def _fetch(self) -> Mapping[str, Any]:
-        if self.create_etag(self.config.url):
+        if create_etag(data_src=self, url=self.config.url, etag_key="ETag"):
```

Comment:
I am under the impression that fetching and dealing with etags is scattered in many places. Is it?

Reply:
This PR fixes that